Formula/nimbus-eth2.rb

```diff
@@ -0,0 +1,45 @@
+class NimbusEth2 < Formula
+  desc "Nim implementation of the Ethereum 2.0 blockchain"
+  homepage "https://nimbus.guide/"
+  # pull from git tag to get submodules
+  url "https://github.com/status-im/nimbus-eth2.git",
+      tag:      "v1.5.5",
+      revision: "f0f9735955070d297b33fbec466537cb7029e0cc"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
+
+    bin.install "build/nimbus_beacon_node" => "nimbus_beacon_node"
+    bin.install "build/deposit_contract" => "deposit_contract"
+    bin.install "build/ncli" => "ncli"
+    bin.install "build/ncli_db" => "ncli_db"
+    bin.install "build/nimbus_validator_client" => "nimbus_validator_client"
+    bin.install "build/nimbus_signing_node" => "nimbus_signing_node"
```

Comment:
Is there no `make install`?

Reply:
Good idea. Unfortunately not, I've just tried 

`make: *** No rule to make target `install'.  Stop.`

Reply:
What action should we take then? Should I ask upstream to add an `install` target or is `bin.install` okay for now?

Reply:
`bin.install` is fine, but since you're not renaming anything

```rb
    bin.install "build/nimbus_beacon_node"
    bin.install "build/deposit_contract"
    bin.install "build/ncli"
    bin.install "build/ncli_db"
    bin.install "build/nimbus_validator_client"
    bin.install "build/nimbus_signing_node"
```

Reply:
Yep, noticed that too.

This is what I went with

```ruby
    %w[
       nimbus_beacon_node
       deposit_contract
       ncli
       ncli_db
       nimbus_validator_client
       nimbus_signing_node
     ].each do |program|
       bin.install "build/#{program}"
     end
```

Reply:
Some of these binaries are used only by the Nimbus development team for various simulation and testing purposes. I would recommend installing only `nimbus_beacon_node` and `nimbus_validator_client` (and we consider the later alpha quality software at the moment).